### ISSUE TITLE
java.lang.NoSuchFieldError: INSTANCE

### ISSUE THREAD

**Issue by gustavkarlsson** [gustavkarlsson/rocketchat-jira-trigger#45]
After fixing #44 in d22c8c72d3905cfc693a6c4ecb15bb9d12e0410f a new bug was introduced.

This is critical, as the application cannot run at all right now. From the looks of it, it's a classpath issue. 

```
2017-10-25 18:19:48,987 [main] INFO  s.g.r.j.RestClientProvider - Using basic authentication
2017-10-25 18:19:48,992 [main] INFO  s.g.r.j.RestClientProvider - Password provided through configuration
Exception in thread "main" java.lang.NoSuchFieldError: INSTANCE
        at org.apache.http.impl.nio.codecs.DefaultHttpRequestWriterFactory.<init>(DefaultHttpRequestWriterFactory.java:52)
        at org.apache.http.impl.nio.codecs.DefaultHttpRequestWriterFactory.<init>(DefaultHttpRequestWriterFactory.java:56)
        at org.apache.http.impl.nio.codecs.DefaultHttpRequestWriterFactory.<clinit>(DefaultHttpRequestWriterFactory.java:46)
        at org.apache.http.impl.nio.conn.ManagedNHttpClientConnectionFactory.<init>(ManagedNHttpClientConnectionFactory.java:75)
        at org.apache.http.impl.nio.conn.ManagedNHttpClientConnectionFactory.<init>(ManagedNHttpClientConnectionFactory.java:83)
        at org.apache.http.impl.nio.conn.ManagedNHttpClientConnectionFactory.<clinit>(ManagedNHttpClientConnectionFactory.java:64)
        at com.atlassian.httpclient.apache.httpcomponents.ApacheAsyncHttpClient.<init>(ApacheAsyncHttpClient.java:175)
        at com.atlassian.httpclient.apache.httpcomponents.ApacheAsyncHttpClient.<init>(ApacheAsyncHttpClient.java:123)
        at com.atlassian.httpclient.apache.httpcomponents.DefaultHttpClientFactory.doCreate(DefaultHttpClientFactory.java:68)
        at com.atlassian.httpclient.apache.httpcomponents.DefaultHttpClientFactory.create(DefaultHttpClientFactory.java:35)
        at com.atlassian.jira.rest.client.internal.async.AsynchronousHttpClientFactory.createClient(AsynchronousHttpClientFactory.java:63)
        at com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory.create(AsynchronousJiraRestClientFactory.java:35)
        at se.gustavkarlsson.rocketchat.jira_trigger.RestClientProvider.get(RestClientProvider.java:32)
        at se.gustavkarlsson.rocketchat.jira_trigger.Server.<init>(Server.java:48)
        at se.gustavkarlsson.rocketchat.jira_trigger.App.<init>(App.java:19)
        at se.gustavkarlsson.rocketchat.jira_trigger.App.main(App.java:26)
```

**Comment by gustavkarlsson** [comment]
Fixed in [Release 1.2.3](https://github.com/gustavkarlsson/rocketchat-jira-trigger/releases/tag/1.2.3)

Ping @TomaszDom and @pathikb

**Comment by pathikb** [comment]
Thanks @gustavkarlsson, got it to work finally.
